internal/commands/user/disable_test.go

```diff
@@ -0,0 +1,324 @@
+		assert.Equal(t, testUsers[0].ID, cmd.inputs.Users[0])
+		assert.Equal(t, cmd.outputs[0].err, errors.New("client error"))
+		assert.Equal(t, cmd.outputs[0].user, testUsers[0])
+	})
+
+	t.Run("should return an error", func(t *testing.T) {
+		for _, tc := range []struct {
+			description string
+			setupClient func() realm.Client
+			expectedErr error
+		}{
+			{
+				description: "when resolving the app fails",
+				setupClient: func() realm.Client {
+					realmClient := mock.RealmClient{}
+					realmClient.FindAppsFn = func(filter realm.AppFilter) ([]realm.App, error) {
+						return nil, errors.New("something bad happened")
+					}
+					return realmClient
+				},
+				expectedErr: errors.New("something bad happened"),
+			},
+			{
+				description: "when finding the users fails",
+				setupClient: func() realm.Client {
+					realmClient := mock.RealmClient{}
+					realmClient.FindAppsFn = func(filter realm.AppFilter) ([]realm.App, error) {
+						return []realm.App{testApp}, nil
+					}
+					realmClient.FindUsersFn = func(groupID, appID string, filter realm.UserFilter) ([]realm.User, error) {
+						return nil, errors.New("something bad happened")
+					}
+					return realmClient
+				},
+				expectedErr: errors.New("something bad happened"),
+			},
+		} {
+			t.Run(tc.description, func(t *testing.T) {
+				realmClient := tc.setupClient()
+				cmd := &CommandDisable{
+					realmClient: realmClient,
+				}
+				err := cmd.Handler(nil, nil)
+
+				assert.Equal(t, tc.expectedErr, err)
+			})
+		}
+	})
+}
+
+func TestDisableFeedback(t *testing.T) {
+	testUsers := []realm.User{
+		{
+			ID:         "user-1",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
+			Type:       "type-1",
+			Data:       map[string]interface{}{"email": "user-1@test.com"},
+		},
+		{
+			ID:         "user-2",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
+			Type:       "type-2",
+			Data:       map[string]interface{}{"email": "user-2@test.com"},
+		},
+		{
+			ID:         "user-3",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeUserPassword}},
+			Type:       "type-1",
+			Data:       map[string]interface{}{"email": "user-3@test.com"},
+		},
+		{
+			ID:         "user-4",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeAPIKey}},
+			Type:       "type-1",
+			Data:       map[string]interface{}{"name": "name-4"},
+		},
+		{
+			ID:         "user-5",
+			Identities: []realm.UserIdentity{{ProviderType: realm.AuthProviderTypeCustomToken}},
+			Type:       "type-3",
+		},
+	}
+	for _, tc := range []struct {
+		description    string
+		outputs        []userOutput
+		expectedOutput string
+	}{
+		{
+			description:    "should show indicate no users to disable",
+			outputs:        []userOutput{},
+			expectedOutput: "01:23:45 UTC INFO  No users to disable\n",
+		},
+		{
+			description: "should show 1 failed user, command: user disable",
```

Comment:
avoid punctuation in test descriptions (it reads funny in `go test` output) (also the part following the `,` here seems unnecessary?)